### PR TITLE
Testing sightings and targets

### PR DESCRIPTION
**Pull request by denyskulykov**

### CODE REVIEW

File: tests/functional/tests/test_sightings.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+from ctrlibrary.core.utils import get_observables
+from ctrlibrary.threatresponse.enrich import enrich_observe_observables
+
+
+def test_positive_sighting_ip(module_headers):
+    """Perform testing for enrich observe observables endpoint to get
+    sightings for observable from Chronicle Backstory
+
+    ID: CCTRI-
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: ? =)

File: tests/functional/tests/test_sightings.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    sightings = get_observables(
+        response, 'Chronicle Backstory')['data']['sightings']
+    assert sightings['count'] == 100
+
+    for sighting in sightings['docs']:
+        assert sighting['confidence'] == 'High'
+        assert sighting['count']
+        assert sighting['id']
+        assert 'start_time' in sighting['observed_time']
+        assert sighting['schema_version'] == '1.0.16'
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: that value is gonna be dynamical, so remove from code or make one global constant

File: tests/functional/tests/test_sightings.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        assert sighting['type'] == 'sighting'
+
+        assert sighting['source'] == 'Chronicle'
+        assert sighting['title'] == 'Found in Chronicle'
+
+        assert (sighting['observables'] ==
+                [{'value': 'dellsupportcenter.com', 'type': 'ip'}])
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: 1) That value is not ip
2) You looked for `observable = {'type': 'ip', 'value': '1.1.1.1'}` so sighting observables should be the same

File: tests/functional/tests/test_sightings.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    assert sightings['count'] == 100
+
+    for sighting in sightings['docs']:
+        assert sighting['confidence'] == 'High'
+        assert sighting['count']
+        assert sighting['id']
+        assert 'start_time' in sighting['observed_time']
+        assert sighting['schema_version'] == '1.0.16'
+        assert sighting['type'] == 'sighting'
+
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: redundant line

File: tests/functional/tests/test_sightings.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        assert sighting['source'] == 'Chronicle'
+        assert sighting['title'] == 'Found in Chronicle'
+
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: redundant line

File: tests/functional/tests/test_sightings.py [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+            assert target['type'] == 'endpoint'
+            assert target['observables'][0]['type'] == 'hostname'
+            assert 'start_time' in target['observed_time']
+
+
+def test_positive_sighting_domain(module_headers):
+    """Perform testing for enrich observe observables endpoint to get
+    sightings for observable from Chronicle Backstory
+
+    ID: CCTRI-
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: ?

File: tests/functional/tests/test_sightings.py [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+                [{'value': 'dellsupportcenter.com', 'type': 'ip'}])
+
+        for target in sighting['targets']:
+            assert target['type'] == 'endpoint'
+            assert target['observables'][0]['type'] == 'hostname'
+            assert 'start_time' in target['observed_time']
+
+
+def test_positive_sighting_domain(module_headers):
+    """Perform testing for enrich observe observables endpoint to get
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: Both tests contain same description, please change it to reflect information  that you provided in signature of the test

File: tests/functional/tests/test_sightings.py [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    sightings = get_observables(
+        response['data'], 'Chronicle Backstory')['data']['sightings']
+    assert sightings['count'] == 100
+
+    for sighting in sightings['docs']:
+        assert sighting['confidence'] == 'High'
+        assert sighting['count']
+        assert sighting['id']
+        assert 'start_time' in sighting['observed_time']
+        assert sighting['schema_version'] == '1.0.16'
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: same

File: tests/functional/tests/test_sightings.py [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+        assert sighting['type'] == 'sighting'
+
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: same

File: tests/functional/tests/test_sightings.py [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+        assert sighting['source'] == 'Chronicle'
+        assert sighting['title'] == 'Found in Chronicle'
+
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: same

File: tests/functional/tests/test_sightings.py (hunk 2 above)
Comment (denyskulykov) [reply tests/functional/tests/test_sightings.py]: I leave just checking that dict has key schema_version

File: tests/functional/tests/test_sightings.py (hunk 3 above)
Comment (denyskulykov) [reply tests/functional/tests/test_sightings.py]: Created Bug - https://jira.softserveinc.com/browse/CCTRI-905

File: tests/functional/tests/test_sightings.py [hunk 11]
```diff
@@ -31,14 +31,13 @@ def test_positive_sighting_ip(module_headers):
...
+        assert 'schema_version' in sighting
         assert sighting['type'] == 'sighting'
-
         assert sighting['source'] == 'Chronicle'
         assert sighting['title'] == 'Found in Chronicle'
-
+        # Create Bug - https://jira.softserveinc.com/browse/CCTRI-905
         assert (sighting['observables'] ==
-                [{'value': 'dellsupportcenter.com', 'type': 'ip'}])
+                [{'value': 'dellsupportcenter.com', 'type': 'domain'}])
```
Comment (oshtaier) [inline tests/functional/tests/test_sightings.py]: in bug you have
```
Expected result:
{'type': 'ip', 'value': '1.1.1.1'}

Actual result:
{'value': 'dellsupportcenter.com', 'type': 'ip'}
```

so change that line on `{'type': 'ip', 'value': '1.1.1.1'}` if it is expected

**Pull request merged**